CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-## 4.0.1
+## 5.0.0
+- Added support for new TLD's ( .888, .nft, .coin, .blockchain, .wallet, .x, .bitcoin, .dao )
+- Replace CNS with UNS
+- Rename Eip1993Factories -> Eip1193Factories
+- Introduce ZilliqaProvider for ZNS
+- Add new method `Resolution#registryAddress(domain: string): Promise<string>` 
+- Return Ens support
+- Updated network-config
+- Add custom network support for ENS
+- Revert @zilliqa-js/core version that supports node version 10
```

Comment:
@JohnnyJumper 

```suggestion
- Revert @zilliqa-js/core version that supports node version 10
```

Why we did this change?

Reply:
I am unsure. I took this change from commit description 5c0398c450cb4ca6bdbdd2a50fc2ca0c6656f529

Reply:
@JohnnyJumper Please be careful when you putting changes in changelog. Commit history can't be a source of truth when you getting a list of changes. For example - we haven't had zilliqa-js core dependency at all, take a look there https://github.com/unstoppabledomains/resolution/pull/166/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R81

The only source of truth you can use is PR scope and file changes compared with master branch